%!TEX root = root.tex

\chapter{Introduction}
\label{chap:intro}
\chaptermark{Introduction}

Cancer is a disease defined by aberrant proliferation of cells that have acquired invasiveness into surrounding tissues of the human body \cite{RN21}. As a whole, cancer is estimated to have caused 600,000 deaths in the United States in 2017 \cite{RN165}. The biological process of cancer development has been associated with numerous hallmarks known to circumvent the otherwise restricted growth of a normal cell \cite{RN24}. The ongoing effort to reduce cancer mortality, whether by prevention or new treatments, may require a deeper understanding of the processes that lead to the development and progression of cancer. Given the limited throughput to study human cancers experimentally \cite{RN61}, my dissertation is focused on developing new computational methods to identify mutational drivers of cancer from the big data arising through large-scale DNA sequencing. Specifically, I will analyze protein-coding mutations that happen somatically, i.e., starting from embryogenesis, mutations that occur in the cells of the body (excluding germ cells), and therefore are not inherited.

\section{Cancer as a genetic disease}
\label{sec:section}

Cancer's foundation as a genetic disease was first proposed more than a century ago by observations of cells with chromosomal aberrations \cite{RN18}. There was only sparse support for this hypothesis until the observation that chicken cells contained a homologous sequence to a gene in a known cancer-related virus, avian sarcoma virus, \cite{RN20} and, further, that a single nucleotide change at codon 12 of the human gene \textit{HRAS} could oncogenically transform bladder cells \cite{RN21, RN19}. Endogenous human genes, when mutated, could therefore contribute to the growth of cancer. As a technical note, I will refer to such genes that contain mutations which increase the net growth of cells toward cancer as \q{cancer driver genes}. However, it was not clear at the time whether all such cancer driver genes would fit the mold of \textit{HRAS}.  Now it is understood that cancer driver genes fall into two broad categories, oncogenes and tumor suppressor genes. Oncogenes, like \textit{HRAS}, acquire mutations that generate gain-of-function, while tumor suppressor genes acquire mutations that cause loss of function. Originally the view of tumor suppressor genes was as biallelic loss-of-function of both gene copies (the \q{two-hit hypothesis}\cite{RN22}), such as by the combined effect of a deletion (or loss-of-heterozygosity) and a mutation, like \textit{RB1} in retinoblastoma \cite{RN23} and \texit{TP53} in colorectal cancer \cite{RN1}. However, for some tumor suppressor genes that are either haploinsufficient or dominant-negative, the mutation of only one copy may be sufficient \cite{RN163, RN161, RN162}.

A particular driver mutation may be neither necessary nor sufficient for the development of cancer. Rather, carcinogenesis, the development of cancer, often is a multi-step process (estimated 2-8 \cite{RN25}) involving several driver mutations, where the combined effect of multiple mutations is sufficient. In the case of colorectal cancer, it is estimated 3 mutational drivers are required \cite{RN27}. The driver mutation at each step causes a clonal expansion of cells because of their selective growth advantage; thus, leading to progression from a small adenoma to a large adenoma and eventually to a carcinoma in colorectal cancers \cite{RN26}. As an example, a particular cancer's sequence of driver mutations could initiate with an \textit{APC} gene mutation followed by a \textit{KRAS} mutation and subsequent \textit{TP53} mutations. But particular driver mutations are not necessarily exclusive to each stage that leads to colorectal cancer \cite{RN26}. Moreover, in another patient's cancer, driver mutations in different genes could also lead to colorectal cancer \cite{RN48}. Lastly, even within a single tumor, there may be multiple competing sub-clones with different compositions of driver mutations (termed \q{intra-tumor heterogeneity}).  Carcinogenesis, therefore, is not a simple fixed linear path of driver mutations, but instead a remarkably heterogeneous mix of multiple possible paths. 

Only in the past decade have improvements in DNA sequencing technology made cancer sequencing studies feasible for cataloging large numbers of mutations in human cancers. The first wave of cancer sequencing studies \cite{RN2, RN4, RN3} analyzed common cancers, such as breast and colorectal cancers, and sequenced only targeted portions of the exome (the regions encoding genes). Due to technical limitations and prohibitive cost, they employed a Discovery-Validation study design where mutations were first detected more comprehensively in a smaller number of samples but then validated against a larger set of samples. Although soon after, milestone studies would sequence the whole-exome of pancreatic cancers \cite{RN5} and glioblastoma multiforme \cite{RN6}.  Also, in the same year, the first pilot project of the The Cancer Genome Atlas (TCGA) analyzed glioblastoma multiforme \cite{RN7}, the beginning of a consortium that would analyze thousands of human cancers in the coming years. The genomic breadth of sequencing was expanded by several whole-genome sequencing studies analyzing a few samples in leukemia, lung cancer, and melanoma \cite{RN8, RN9, RN10}. By 2011 the TCGA analyzed 316 ovarian carcinoma samples by whole-exome sequencing \cite{RN11}, which started to reach the large sample sizes necessary for statistically implicating cancer drivers.

\section{Positive selection and the statistical identification of cancer drivers}
\label{sec:section}

Cancer sequencing studies quickly made it evident that driver mutations only constitute a small percentage of the potential 100's or 1,000's of mutations observed with a single exome \cite{RN25}. The major question is therefore not what mutations are detected in cancer but, rather, which mutations are drivers as opposed to \q{passengers} that do not contribute to tumorigenesis? Because the vast majority of mutations are passengers, it is difficult to distinguish a driver mutation from many passenger mutations; also, like a driver mutation, many passenger mutations may be completely clonal in a particular cancer sample, because they happened before the founding clone's driver mutation \cite{RN160, RN50}. Passenger mutations effectively hitchhike off of the selective growth advantage provided by driver mutations. The key distinction is, when considered across many cancer samples, that driver mutations are positively selected for in cancer and therefore should be disproportionately represented. Computational methods have therefore evaluated signals of positive selection in cancer at various scales, including at the protein, domain/region, and mutation level \cite{RN52}. Although the precise way positive selection is statistically measured varies \cite{RN49}, the essence is to analyze patterns of mutations across many samples and rule out the possibility that the mutations are explainable by a random background accumulation of mutations alone. 

As a consequence, identifying positive selection of driver mutations also requires understanding the converse, how passenger mutations accumulate in cancer. Passenger mutations have only \textless1\% of non-silent mutations eliminated by negative selection, indicating that most of the variability in the number of passenger mutations is due to mutation rates rather than selection \cite{RN56}. A simple model is that passenger mutations accumulate at a universal background rate per base across the genome \cite{RN3}, after adjustments for the nucleotide sequence context. This model fails to capture key mutational processes in cancer, such as the background mutation rate varying by over two orders of magnitudes between cancer types \cite{RN13} and also varying patient-by-patient, especially when a tumor has defective DNA damage repair or mutagen exposure \cite{RN51}. Moreover, regional variation within the genome of replication timing, gene expression, and chromatin structure leads to $\sim$3-fold differences in the background mutation rate \cite{RN13}. In certain cancers, kataegis causes genomically localized hypermutation \cite{RN164}. Accurate models therefore need to account for the greater dispersion caused by this mutational heterogeneity.

A focus for many large cancer sequencing studies has been to identify cancer driver genes, usually done in one of three ways. The most common approach, which I term as a significantly mutated gene method, has been to compare the number of mutations within a gene to that expected by a background mutation rate \cite{RN158, RN156, RN2, RN3}.  The accuracy of the background mutation rate, therefore, becomes the critical parameter to estimate. Recent improvements in estimating regional variation in mutation rate across the genome lead to reduced false positives in the method MutSigCV \cite{RN55, RN29}.  The second approach, functional impact bias, evaluates whether a gene contains mutations that are skewed towards higher predicted impact \cite{RN53}. The score of a mutation usually reflects either evolutionary conservation of the protein sequence or machine learning methods that predict the deleteriousness of a variant. Lastly, evaluating the positional clustering of mutations has also been used at the sequence- and structure-level of a protein \cite{RN133, RN16, RN55, RN60, RN62, RN131, RN132, RN155, RN45, RN46, RN110, RN87, RN151, RN152, RN15, RN54}. However, not all tumor suppressor genes may exhibit mutational clustering, and therefore these methods may be better at identifying oncogenes due to their highly localized activating mutations.

Although a cancer driver gene, by definition, contains a driver mutation, not all mutations within a cancer driver gene are necessarily cancer drivers. Especially considering the large size of genes, passenger mutations will be observed in large numbers when analyzing many cancer samples \cite{RN56}. To address this issue, recent methods, known as hotspot mutation detection, have therefore focused on smaller regions, such as protein domains \cite{RN54}, protein-protein interfaces \cite{RN53}, and individual codons \cite{RN23}.

However, if driver mutations are in separate locations of the protein sequence, hotspot detection based on protein sequence may lose statistical power. Often this results from residues being far apart in protein sequence but actually proximal in the folded protein structure. Since there is a relationship between protein structure and function \cite{RN112, RN113}, I and others have developed computational methods for hotspot detection in 3D space of protein structure \cite{RN133, RN60, RN131, RN132, RN45, RN105, RN151, RN152, RN15}. Hotspot detection in protein structure has the advantage of generating plausible hypotheses about the function of the mutation given the spatial proximity to known functional sites in the protein \cite{RN57, RN60}.

Cancer driver prediction at the level of individual mutations has largely focused on missense mutations, the most common type of protein-coding mutation in cancer \cite{RN25}. Typically, machine learning approaches have been used to leverage features characterizing mutations. Although features vary substantially by method \cite{RN32, RN33, RN35, RN36, RN39, RN29, RN37}, they usually include inter-species evolutionary conservation of the protein sequence, features of the local protein environment, molecular function annotations, and biophysical characterizations of the amino acid substitution. Cancer-focused machine learning methods have previously tried to enhance performance by training cancer type-specific models \cite{RN10, RN12} or boosting data with synthetic passenger missense mutations \cite{RN10}. Despite the capability of utilizing many features, with the exception of a few gene-level features in ParsSNP \cite{RN9}, machine learning methods typically have not used mutational patterns characterizing the genetic variation observed in human cancers. Furthermore, a systematic comparative study of 15 methods concluded that none of them were sufficiently reliable for experimental or clinical follow-through \cite{RN46}. I, and others, have hypothesized that determining the impact of missense mutations requires proper context \cite{RN47, RN57}, which have not been sufficiently leveraged in a comprehensive manner from the current generation of methods. Context includes both prior knowledge about the functional importance of genes or gene subregions in which a mutation occurs, and mutational patterns that are now evident from cancer sequencing studies of many thousands of patients.

\section{Large-scale cancer driver discovery}
\label{sec:section}

The application of computational methods to identify mutational drivers of cancer has expanded with the growth in sample size of cancer sequencing studies \cite{RN99, RN105, RN14, RN87, RN158, RN12, RN13, RN54, RN96, RN98, RN154, RN43}. A comprehensive analysis of 3,281 cancers comprising 12 cancer types from the TCGA revealed 125 associated cancer driver genes \cite{RN12}.  A subsequent study identified 224 cancer driver genes in 21 cancer types, and further suggested by sub-sampling analysis that the discovery of new cancer driver genes does not show evidence of saturation at current sample sizes \cite{RN14}. Combined with the low mutation frequency of many identified cancer driver genes, it has been hypothesized there is a \q{long tail} of drivers of increasing rarity, with more cancer drivers on the horizon \cite{RN148, RN147}. Prior statistical power calculations suggest this is reasonable given driver genes estimated at a 2\% frequency of cancers may require sample size as high as 5,000 for certain cancer types with high mutation rate and, in total across all cancer types, 100,000 sequenced cancer samples may be needed \cite{RN14}. This is well above the number of samples per cancer type available in The Cancer Genome Atlas. 

Only a relatively few studies have started to focus on identifying cancer drivers at sub-gene resolution. Initially, studies focused on identifying protein domains \cite{RN46, RN110}. More recently, studies have progressed to varying sized hotspots and towards codon-level resolution \cite{RN166, RN16, RN55}. However, I and others have noted such approaches currently are biased towards finding hotspot regions in oncogenes as opposed to tumor suppressor genes. This is a result of tumor suppressor genes having loss-of-function driver mutations that are more diverse and spread over a larger region of the protein \cite{RN60}. 

\section{Relevance to current study}
\label{sec:section}

Ultimately, computational methods have progressively sought to understand whether a particular mutation in a patient's cancer is a cancer driver mutation. However, given current approaches may require 100,000 cancer samples to just identify cancer driver genes, identifying particular driver mutations within those genes may require an even greater number of sequenced cancers. Due to the millions of mutations currently being identified in human cancers, laborious experimental validation of all mutations is not feasible because of lack of throughput \cite{RN57, RN61}. A more statistically powerful computational method would be greatly beneficial to the cancer genomics community; enabling improved utilization of cancer sequencing studies. Insight into cancer driver mutations can be of substantial clinical relevance, such as indicators of prognosis \cite{RN6, RN159}, therapeutic response \cite{RN58}, drug targets \cite{RN104}, and as biomarkers for early detection of cancer \cite{RN59}. 

My dissertation covers four primary aims: \textbf{(1)} the modeling of somatic mutations, \textbf{(2)} development of new computational methods, \textbf{(3)} benchmarking of computational predictions, and \textbf{(4)} systematic driver discovery across thousands of human cancer samples. Where pertinent, results of methodological benchmarks (3) and systematic driver discovery (4) are combined in the same chapter as the corresponding developed computational method (2).

Although there are many existing computational methods, I find that prior methods have not yet adequately combined multiple signals of positive selection of driver mutations in cancer. I hypothesize better characterization of cancer drivers, particularly those that occur at relatively low prevalence, can be obtained by a carefully designed machine learning approach, which leverages mutational patterns in cancer sequencing studies that are characteristic of oncogenes and tumor suppressor genes. Moreover, I show that combining multiple scales of information - including at the gene, region, and mutation level - has substantial benefits.

It has been difficult to evaluate progress in this area because many published methods do not rigorously compare their relative merits to those developed by others. I establish extensive benchmarks for cancer driver prediction to address this shortcoming. Importantly, I develop novel metrics to assess the current landscape of predictions, which addresses the need for rigorous evaluation criteria given the lack of a true gold standard for predicting cancer drivers. My analysis points to the strengths and weaknesses of each of the currently available methods and offers guidance for improving them in the future.

The recent completion of The Cancer Genome Atlas has provided a unique capability to understand cancer at an unprecedented scale. Here, I predicted protein-coding driver mutations in nearly 10,000 cancers and characterize the landscape of drivers across human cancers. This involves interrogating fundamental questions regarding cancer and driver mutations, such as their cancer type specificity, commonness or rarity, the balance and characteristics of oncogenes and tumor suppressor genes, and the likely future trajectory of cancer driver discovery.




