%!TEX root = root.tex

\chapter{Concluding remarks}
\label{chap:ch8}
\chaptermark{Concluding remarks}

The first confirmed human cancer driver gene, HRAS, was identified in 1982 \cite{RN21, RN19}. The past decade has seen the list of likely cancer driver genes grow rapidly. Although partly reflecting the growth in size of studies due to advances in next generation sequencing, it also reflects improvements in computational techniques. Computational methods are starting to become more robust with realistic models of how somatic mutations accumulate in cancer. Moreover, studies are now moving to understanding cancer drivers at increasing resolution -- moving from genes to individual mutations.

The first part of my dissertation (Chapter 2) focused on how to appropriately statistically model the accumulation of somatic mutations in cancer. The typical choice of modeling the background mutation rate is problematic because it is highly variable at multiple scales. However, a key insight is that covariates usually modulate mutation rate at the scale of megabases within the genome, but nearly all genes span $<$1MB. By statistically conditioning on the total number of mutations within a gene while simulating mutations, nuisance factors influencing mutation rate, which are not always measured or known, are substantially lessened. This approach allows substantial flexibility in comprehensively modeling the many mutational patterns indicative of positive selection in cancer.

Here, I introduced several new computational methods to analyze cancer drivers at different levels -- such as the gene (20/20+, Chapter 3), region (HotMAPS, Chapter 5), and mutation (CHASMplus, Chapter 6). I used these methods to interrogate fundamental questions regarding cancer driver mutations, such as their cancer type specificity, commonness or rarity, the balance and characteristics of oncogenes and tumor suppressor genes, and the likely future trajectory of cancer driver discovery. 20/20+ identified that the balance of oncogenes and tumor suppressor genes (TSG) varies considerably by cancer type, some having all TSGs while others having mostly oncogenes. Also, CHASMplus found significantly more rare cancer driver mutations than previously understood, which is supportive of the long-tail hypothesis. The high prevalence of rare driver mutations suggests interpretation of a cancer genome will need to be increasingly personalized, since a patientâ€™s driver mutation may have not been previously observed. 

Due to the lack of a gold-standard for cancer drivers, I also developed a benchmark for cancer driver gene prediction (Chapter 4). This included five components: number of significant genes, overlap with previous literature, overlap with other methods, divergence of p-values from expectation, and consistency. I found that some methods did not accurately model the heterogeneity of accumulation of mutations by chance. As new computational methods are developed, it will be critical to effectively benchmark them against existing state-of-the-art.

I also used computational methods in an attempt to comprehensively discover driver genes and mutations in nearly 10,000 human cancers (Chapter 7). This was done as a consensus across institutions from The Cancer Genome Atlas, and by extension a consensus of computational methods. This idea of a consensus is not new, and has been used successfully in many other domains \cite{RN190, RN189}. The analysis revealed 299 cancer driver genes across 33 cancer types. Many of the cancer driver genes were cancer type specific, but others were found in multiple cancer types that had a common cell of origin. Analysis also found ~3,400 unique missense mutations as likely cancer drivers, with high validation rates compared to an in vitro assay. The results from CHASMplus, however, suggest that in some cancer types the rate of discovery is starting to exhibit diminishing returns.

Although the landscape of cancer drivers in primary tumors are progressively getting more fully explored, there remains many aspects that are still poorly understood. Sequencing untreated primary tumors gives an understanding of one time point in the natural evolutionary history of cancers. Understanding the full heterogeneity and dynamics of cancer will require sequencing both before (pre-cancerous lesions) and after (metastases). This will provide more understanding of such questions as: do cancers need particular gatekeeper drivers to initiate tumorigenesis?, and how much does the temporal ordering of driver mutations matter as opposed to overall driver mutation burden? In addition, the role of drivers in the non-coding region of the genome is only beginning to be characterized, but early studies indicate that $>$90\% of driver point mutations may actually reside in coding regions \cite{RN17}. The discrepancy of why other common diseases are estimated to have the majority of heritability in non-coding regions \cite{RN192, RN193} remains to be understood.

A complete catalog of all cancer drivers, in it of it self, will not suffice. Cancer driver mutations will need to be related to their functional consequence, interaction with the microenvironment, and to other driver mutations. Given the prevalence of rare driver mutations, the scale of experiments designed to functionally characterize or validate potential driver mutations will need to match pace. On the basis of this, a mechanistic insight will be critical for a rational understanding of the effects of targeted drugs and optimization of drug combinations. Lastly, driver mutations may not exert their effect in isolation, but rather epistatically interact with other driver mutations. For instance, co-mutations of \textit{KRAS}, \textit{ATM}, \textit{STK11}, and \textit{KEAP1} in lung adenocarcinoma define a cancer subtype with different biology and immune signatures \cite{RN191}.

In summary, cancer was first understood as a genetic disease by observing large changes in chromosomes through a microscope \cite{RN18}. Now, computational methods, like those developed here, are serving as a mathematical microscope into understanding driver mutations in cancer. The tools from statistics allow control of false discoveries, which prevents errant mistakes. While machine learning translates many biological features indicative of positive selection into concrete predictions of driver mutations. The convergence of this and large-scale cancer sequencing has turned many aspects of cancer research into a data science. As future research focuses on greater precision required for interpreting individual mutations in a patient's cancer, the trend of data science in cancer research will likely continue.
