%!TEX root = root.tex

\begin{vita}
\ssp

\noindent\begin{large}\textbf{Collin Tokheim}\end{large}\\
116 W University Pkwy, APT 1226 \\
Baltimore, MD, 21210 \\
ctokheim@jhu.edu \\
\vspace{5pt}
\hrule
\vspace{10pt}
\vskip .1in
%\hrulefill 

\noindent\begin{large}\textbf{EDUCATION}\end{large}
\vskip .1in
\noindent Johns Hopkins University \\
Ph.D., Biomedical Engineering, September 2018 \\
GPA 4.0/4.0 \\

\noindent University of Iowa \\
BSE, Biomedical Engineering, 2011 \\
GPA 4.0/4.0 \\

\noindent\begin{large}\textbf{RESEARCH EXPERIENCE}\end{large}
\vskip .1in
\noindent\textbf{Graduate research assistant, Johns Hopkins University (2014-2018)}\\
\textit{Advisor: Dr. Rachel Karchin, Biomedical Engineering and Oncology}\\
Topic: Statistical identification of mutational drivers in human cancers through development of cutting-edge computational methods\\

\noindent\textbf{Staff research associate, University of California, Los Angeles (1/2013-6/2013)}\\
\textit{Advisor: Dr. Yi Xing, Microbiology, Immunology, \& Molecular Genetics}\\
Topic: Genomic analysis of the transcriptome with emphasis on alternative splicing\\

\noindent\textbf{Research Assistant, University of Iowa (4/2011-12/2012)}\\
\textit{Advisor: Dr. Yi Xing, Microbiology, Immunology, \& Molecular Genetics}\\
Topic: Genomic analysis of the transcriptome with emphasis on alternative splicing\\

\noindent\textbf{Undergraduate Research Assistant, University of Iowa (4/2010-8/2011)}\\
\textit{Advisor: Dr. Xiaodong Wu, Electrical and Computer Engineering and Radiation Oncology}\\
Topic: Computational analysis of computed tomography (CT) scans of tumors.\\

\noindent\begin{large}\textbf{DISTINCTIONS, AWARDS, \& HONORS}\end{large}
\vskip .1in
\begin{itemize}
  \itemsep0em
  \item Martin and Carol Macht Award (2018)
  \item First author PNAS paper featured in Baltimore Sun newspaper (Jan 2017)
  \item National Cancer Institute (NCI) NRSA Fellowship F31CA200266 (2015-present)
  \item Alpha Eta Mu Beta Biomedical Engineering Honor Society (2016-present)
  \item Top performing team in the Personal Genome Project challenge in Critical Assessment of Genome Interpretation (CAGI) competition (Mar 2016)
  \item Graduated with Highest Distinction (May 2011)
  \item Tau Beta Pi Engineering Honor Society (2010-present) 
  \item William C. Blackburn Engineering Scholarship (2009-2011)
  \item University of Iowa President's List (2008-2011) 
  \item E. Lester and Frances M. Williams Fund Scholarship (2008-2009)
  \item College of Engineering Dean's List (2007-2011)
  \item Engineering Excellence Scholarship (2007-2008)
\end{itemize}
\vskip .15in

\noindent\begin{large}\textbf{PEER-REVIEWED PUBLICATIONS}\end{large}
\vskip .1in
\noindent(Google Scholar H-index = 7, total citations = 295)\\
* co-first author
\vskip .1in

Matthew H. Bailey*, \underline{Collin Tokheim}*, Eduard Porta-Pardo*, Sohini Sengupta, Denis Bertrand, Amila Weerasinghe, Antonio Colaprico, Michael C. Wendl, Jaegil Kim,  Brendan Reardon, Kwok-Shing Ng, Kang Jin Jeong, Song Cao, Zixing Wang, JianJiong Gao, Qingsong Gao, Fang Wang, Eric Minwei Liu, Loris Mularoni, Carlota Rubio-Perez, Niranjan Nagarajan, Isidro Cort√©s-Ciriano, Daniel Cui Zhou, Wen-Wei Liang, Julian M. Hess, Venkata D. Yellapantula, David Tamborero, Abel Gonzalez-Perez, Chayaporn Suphavilai, Jia Yu Ko, Ekta Khurana, Peter J. Park, Eliezer Van Allen, Han Liang, The MC Working Group, The Cancer Genome Atlas Research Network, Michael Lawrence, Adam Godzik, Nuria Lopez-Bigas, Josh Stuart, David Wheeler, Gad Getz, Ken Chen, Alexander J. Lazar, Gordon B Mills, Rachel Karchin, Li Ding. Comprehensive Discovery and Characterization of Driver Genes and Mutations in Cancer. Cell. In Press.\\

\underline{Tokheim, C. J.}, Papadopoulos, N., Kinzler, K. W., Vogelstein, B. \& Karchin, R. Evaluating the evaluation of cancer driver genes. Proc Natl Acad Sci U S A 113, 14330-14335, doi:10.1073/pnas.1616440113 (2016).\\

\underline{Tokheim, C.}, Bhattacharya, R., Niknafs, N., Gygax, D. M., Kim, R., Ryan, M., Masica, D. L. \& Karchin, R. Exome-Scale Discovery of Hotspot Mutation Regions in Human Cancer Using 3D Protein Structure. Cancer Res 76, 3719-3731, doi:10.1158/0008-5472.CAN-15-3190 (2016).\\

Bertotti, A., Papp, E., Jones, S., Adleff, V., Anagnostou, V., Lupo, B., Sausen, M., Phallen, J., Hruban, C. A., \underline{Tokheim, C.}, Niknafs, N., Nesselbush, M., Lytle, K., Sassi, F., Cottino, F., Migliardi, G., Zanella, E. R., Ribero, D., Russolillo, N., Mellano, A., Muratore, A., Paraluppi, G., Salizzoni, M., Marsoni, S., Kragh, M., Lantto, J., Cassingena, A., Li, Q. K., Karchin, R., Scharpf, R., Sartore-Bianchi, A., Siena, S., Diaz, L. A., Jr., Trusolino, L. \& Velculescu, V. E. The genomic landscape of response to EGFR blockade in colorectal cancer. Nature 526, 263-267, doi:10.1038/nature14969 (2015).\\

Patrick Kwok-Shing Ng, J. L., Kang Jin Jeong, Shan Shao, Hu Chen, Yiu Huen Tsang, Sohini Sengupta, Zixing Wang, Venkata Hemanjani Bhavana, Richard Tran, Stephanie Soewito, Darlan Conterno Minussi, Daniela Moreno, Kathleen Kong, Turgut Dogruluk, Hengyu Lu, Jianjiong Gao, \underline{Collin Tokheim}, Daniel Cui Zhou, Jia Zeng, Carman Ka Man Ip, Zhenlin Ju, Matthew Wester, Shuangxing Yu, Yongsheng Li, Christopher Vellano, Nikolaus Schultz, Rachel Karchin, Li Ding, Yiling Lu, Lydia Wai Ting Cheung, Ken Chen,  Kenna R. Shaw, Funda Meric-Bernstam, Kenneth L. Scott, Song Yi, Nidhi Sahni, Han Liang, and Gordon B. Mills. Systematic Functional Annotation of Somatic Mutations in Cancer. Cancer Cell. In press.\\

Chung, C. H., Guthrie, V. B., Masica, D. L., \underline{Tokheim, C.}, Kang, H., Richmon, J., Agrawal, N., Fakhry, C., Quon, H., Subramaniam, R. M., Zuo, Z., Seiwert, T., Chalmers, Z. R., Frampton, G. M., Ali, S. M., Yelensky, R., Stephens, P. J., Miller, V. A., Karchin, R. \& Bishop, J. A. Genomic alterations in head and neck squamous cell carcinoma determined by cancer gene-targeted sequencing. Ann Oncol 26, 1216-1223, doi:10.1093/annonc/mdv109 (2015).\\

Li Ding, M. H. B., Eduard Porta-Pardo, Vesteinn Thorsson, Antonio Colaprico, Denis Bertrand, David L. Gibbs, Amila Weerasinghe, Kuan-lin Huang, Matthew A Wyczalkowski, Sam Q. Sun, \underline{Collin Tokheim}, Reyka Jayasinghe, Feng Chen, Lihua Yu, Alison M Taylor, Andrew D Cherniack, Jaegil Kim, Rehan Akbani, Chayaporn Suphavilai, Niranjan Nagarajan, Joshua M. Stuart, Gordon B Mills, Benjamin Vincent, Carolyn M Hutter, Jean Claude Zenklusen, Katherine A Hoadley, Michael C. Wendl, llya Shmulevich, Alexander J. Lazar, David Wheeler, Gad Getz, The Cancer Genome Atlas Research Network. Perspective on Central Oncogenic Processes at the End of the Beginning of Cancer Genomics. [Submitted]\\

\underline{Tokheim, C.}, Park, J. W. \& Xing, Y. PrimerSeq: Design and visualization of RT-PCR primers for alternative splicing using RNA-seq data. Genomics Proteomics Bioinformatics 12, 105-109, doi:10.1016/j.gpb.2014.04.001 (2014).\\

Masica, D. L., Douville, C., \underline{Tokheim, C.}, Bhattacharya, R., Kim, R., Moad, K., Ryan, M. C. \& Karchin, R. CRAVAT 4: Cancer-Related Analysis of Variants Toolkit. Cancer Res 77, e35-e38, doi:10.1158/0008-5472.CAN-17-0338 (2017).\\

Park, J. W., \underline{Tokheim, C.}, Shen, S. \& Xing, Y. Identifying differential alternative splicing events from RNA sequencing data using RNASeq-MATS. Methods Mol Biol 1038, 171-179, doi:10.1007/978-1-62703-514-9\_10 (2013).\\

Wang, J., Lu, Z. X., \underline{Tokheim, C. J.}, Miller, S. E. \& Xing, Y. Species-specific exon loss in human transcriptomes. Mol Biol Evol 32, 481-494, doi:10.1093/ molbev/msu317 (2015). \\

Lu, Z. X., Huang, Q., Park, J. W., Shen, S., Lin, L., \underline{Tokheim, C. J.}, Henry, M. D. \& Xing, Y. Transcriptome-wide landscape of pre-mRNA alternative splicing associated with metastatic colonization. Mol Cancer Res 13, 305-318, doi:10.1158/ 1541-7786.MCR-14-0366 (2015). \\

Rettig, E. M., Talbot, C. C., Jr., Sausen, M., Jones, S., Bishop, J. A., Wood, L. D., \underline{Tokheim, C.}, Niknafs, N., Karchin, R., Fertig, E. J., Wheelan, S. J., Marchionni, L., Considine, M., Fakhry, C., Papadopoulos, N., Kinzler, K. W., Vogelstein, B., Ha, P. K. \& Agrawal, N. Whole-Genome Sequencing of Salivary Gland Adenoid Cystic Carcinoma. Cancer Prev Res (Phila) 9, 265-274, doi:10.1158/1940-6207.CAPR-15-0316 (2016). \\

Cai, B., Li, B., Kiga, N., Thusberg, J., Bergquist, T., Chen, Y. C., Niknafs, N., Carter, H., \underline{Tokheim, C.}, Beleva-Guthrie, V., Douville, C., Bhattacharya, R., Yeo, H. T. G., Fan, J., Sengupta, S., Kim, D., Cline, M., Turner, T., Diekhans, M., Zaucha, J., Pal, L. R., Cao, C., Yu, C. H., Yin, Y., Carraro, M., Giollo, M., Ferrari, C., Leonardi, E., Tosatto, S. C. E., Bobe, J., Ball, M., Hoskins, R. A., Repo, S., Church, G., Brenner, S. E., Moult, J., Gough, J., Stanke, M., Karchin, R. \& Mooney, S. D. Matching phenotypes to whole genomes: Lessons learned from four iterations of the personal genome project community challenges. Hum Mutat 38, 1266-1276, doi:10.1002/humu.23265 (2017). \\
\vskip .1in
\noindent\begin{large}\textbf{RESEARCH PRESENTATIONS \& CONFERENCES}\end{large}
\vskip .1in
\noindent Posters
\begin{itemize}
  \itemsep0em
  \item American Society of Human Genetics Annual Meeting (10/2016). Title: \q{CRAVAT 4.3: informatics tools for high-throughput analysis of exome variants}
  \item GRC SNPs \& Disease (6/2016). Title: \q{HotMAPS \& MuPiT: Somatic missense mutation clustering in protein structures aids cancer driver discovery and interpretation}
  \item Critical Assessment of Genome Interpretation conference (3/2016). Title: \q{Genome Interpretation using Bayesian Inference: Application to Matching Anonymized Genomes to Phenotypic Profiles}
\end{itemize}

\noindent Conferences \& meetings
\begin{itemize}
  \itemsep0em
  \item TCGA PanCanAtlas meeting, Houston (11/2016)
  \item American Society of Human Genetics Annual Meeting, Vancouver (10/2016)
  \item Critical Assessment of Genome Interpretation conference, San Francisco (3/2016)
  \item TCGA PanCanAtlas meeting, Santa Cruz (2/2016)
  \item American Association of Cancer Research Annual Meeting, Philadelphia (4/2015)
  \item Biology of Genomes Conference, Cold Spring Harbor (5/2014)
\end{itemize}
\vskip .15in
\noindent\begin{large}\textbf{TEACHING EXPERIENCE}\end{large}
\vskip .1in
\noindent Teaching assistant, Foundations of Computational Biology and Bioinformatics II (Spring 2017)\\
Teaching assistant, Systems Bioengineering 3 (Fall 2016)


\end{vita}